modules/objdetect/test/test_arucodetection.cpp

```diff
@@ -613,6 +613,41 @@ TEST(CV_ArucoDetectMarkers, regression_2492)
     }
 }
 
+
+TEST(CV_ArucoDetectMarkers, regression_contour_24220)
+{
+    aruco::ArucoDetector detector;
+    aruco::DetectorParameters detectorParameters = detector.getDetectorParameters();
+    detector.setDetectorParameters(detectorParameters);
```

Comment:
The two lines useless. They do not change detector behavior.

Reply:
removed